src/main/java/org/cactoos/io/EmptyClosableReader.java

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
@rexim I think this class should be either inner class of `ReaderAsBytesTest` or at least stay in `src/test`

Reply:
@yegor256 I agree. `org.cactoos.io` in `src/test` already contains a couple of "Testing Helper Classes" like `SlowInput` or  `SlowInputStream`. I'd go for moving `EmptyClosableReader` there. I have a feeling that this class might be useful for testing similar `Reader` consumers in the future.